### DISCUSSION TITLE
Adding multi tenancy to existing django solution using admin_interface getting error,

### DISCUSSION THREAD

**Discussion by trexmo** [django-tenants/django-tenants#1032] [Q&A]
Adding django_tenants to existing solution and getting the following error on login.  Have created public tenant, domain and new client/domain.

`relation "admin_interface_theme" does not exist LINE 1: ...in_interface_theme"."form_pagination_sticky" FROM "admin_int... ^
`
Was not getting this error before adding django_tenants

**Comment by ethagnawl** [comment]
Did you ever figure this out, @trexmo? I've been banging my head against this for days with nothing to show for it. I was thinking I'd add a route parameter or query parameter and overload the relevant admin methods to set the context. While this sort of works in isolation, Django admin really doesn't want you messing with the URL or query string (my most recent effort) and unsets it on every request.
